angel-data/Web Onboarding NO - Combo.json

```
@@ -0,0 +1,393 @@
+{
+  "id": "5696df77-6e1f-42ff-9585-b3b3ff64ac0c",
+  "ifid": "D3AACABF-A05A-4221-BACB-46035E945B9A",
+  "keywords": {
+    "backButton": "Back",
+    "externalInsuranceProviderConfirmTitle": "",
+    "selectActionSelectLabel": "Select",
+    "tooltipModalInformationLabel": "Information"
```

These need to be translated as well, right?

I guess it's:

```suggestion
    "backButton": "{STORY_KEYWORDS_BACK_BUTTON}",
    "selectActionSelectLabel": "{STORY_KEYWORDS_SELECT_ACTION_LABEL}",
    "tooltipModalInformationLabel": "{STORY_KEYWORDS_TOOLTIP_MODAL_INFORMATION_LABEL}"
```

Just make sure we've added NO translations for them :)

---

Super, thank you